javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,9 @@
+function ExampleConstructor() {}
+console.log('value of ExampleConstructor: ', ExampleConstructor);
+console.log('typeof of ExampleConstructor: ', typeof ExampleConstructor);
```

Comment:
Actually we want to console log value and type of `prototype` property of `ExampleConstructor`